playbook/app/pb_kits/playbook/pb_body/_body.tsx

```diff
@@ -19,7 +19,7 @@ type BodyProps = {
   status?: 'negative' | 'neutral' | 'positive',
   tag?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'span' | 'div',
   text?: string,
-  variant: null | 'link',
+  variant?: null | 'link',
```

Comment:
Do not forget to set the default value as `null`, that might crash something (we hope not).

Reply:
Yeah, already taken care [here](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_body/_body.tsx#L38)